### PR TITLE
Transaction fixes (#997, #1004)

### LINKED ISSUE #997: /send command's amount is not validated before calling eth.sendTransaction
[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

[comment]: # (Describe the feature you would like, or briefly summarise the bug and what you did, what you expected to happen, and what actually happens. Sections below)
*Summary*: 

There is only 1 ETH on account, but `/send 234` passes validation:
v

#### Expected behavior
[comment]: # (Describe what you expected to happen.)

Validation error should appear when a user is trying send `/send` but doesn't have enough ETH on balance. 

#### Actual behavior
[comment]: # (Describe what actually happened.)

`/send` command passes validation successfully regardless account's balance.

### PR DESCRIPTION
**Pull request by alwx**
Fixes #997 and #1004

### CODE REVIEW

File: src/status_im/chat/handlers/input.cljs [hunk 1]
```diff
@@ -212,7 +212,7 @@
   (handlers/side-effect!
     (fn [{:keys [contacts] :as db}
          [_ {{:keys [command metadata args] :as c} :command
-             :keys [message-id chat-id data-type after]}]]
+             :keys [message-id chat-id data-type after] :as a}]]
```
Comment (rasom) [inline src/status_im/chat/handlers/input.cljs]: `a` is not used

Comment (alwx) [inline src/status_im/chat/handlers/input.cljs]: fixed

**Comment by alwx** [comment]
Wait a min, I want to spend more time testing transactions and the way `DiscardTransaction` function works

**Comment by alwx** [comment]
It's finished

**Pull request merged**